doc/omnisharp-vim.txt

```diff
@@ -262,4 +263,11 @@ COMMANDS                                                   *omnisharp-commands*
 :OmniSharpHighlightTypes
     Enable advanced type highlighting for current file
 
+                                                            *:OmniSharpInstall*
+:OmniSharpInstall
+    Download the latest OmniSharp-roslyn binaries and extract them into
+    ~/.omnisharp/omnisharp-roslyn.
+    NOTE: This is currently only supported in Windows in Cygwin/WSL
```

Comment:
I also found that the wording of this might confuse some users into thinking that the command only works on Windows (when using Cygwin or WSL) and not on macOS or Linux

Reply:
Good point, updated